jerry-core/ecma/builtin-objects/ecma-builtin-string-prototype.cpp

```diff
@@ -31,6 +31,10 @@
 #include "jrt-libc-includes.h"
 #include "lit-char-helpers.h"
 
+#ifndef CONFIG_ECMA_COMPACT_PROFILE_DISABLE_REGEXP_BUILTIN
+#include "ecma-regexp-object.h"
+#endif
```

Comment:
Usually, `#endif` is followed by a condition, under which block above is executed.
In the case: `/* !CONFIG_ECMA_COMPACT_PROFILE_DISABLE_REGEXP_BUILTIN */`